extensions/sql-migration/src/api/azure.ts

```diff
@@ -438,8 +438,8 @@ export interface StartDatabaseMigrationRequest {
 			password: string
 		},
 		scope: string,
-		autoCutoverConfiguration?: {
-			autoCutover?: boolean,
+		autoCutoverConfiguration: {
+			autoCutover: boolean,
```

Comment:
We can just change the type to AutoCutoverConfiguration now. 

autoCutoverConfiguration: AutoCutoverConfiguration